manifests/j/JohnMacFarlane/Pandoc/3.6/JohnMacFarlane.Pandoc.installer.yaml

```diff
@@ -0,0 +1,35 @@
+# Created with WinGet Updater using komac v2.8.0
+# yaml-language-server: $schema=https://aka.ms/winget-manifest.installer.1.9.0.schema.json
+
+PackageIdentifier: JohnMacFarlane.Pandoc
+PackageVersion: '3.6'
+InstallerLocale: en-US
+InstallerType: wix
+Scope: user
+InstallModes:
+- interactive
+- silent
+- silentWithProgress
+UpgradeBehavior: uninstallPrevious
+ProductCode: '{9AF3E035-C175-4C5E-B71A-4EA4F890B4F7}'
+ReleaseDate: 2024-12-08
+AppsAndFeaturesEntries:
+- DisplayName: Pandoc 3.6
+  Publisher: John MacFarlane
+  ProductCode: '{9AF3E035-C175-4C5E-B71A-4EA4F890B4F7}'
+  UpgradeCode: '{A68E8EF6-ABB1-4F22-A3C5-68DFDF0AB562}'
+InstallationMetadata:
+  DefaultInstallLocation: '%ProgramFiles%/Pandoc'
+Installers:
+- Architecture: x64

```

Comment:
```suggestion
- Architecture: x64
  Scope: user
```

Reply:
fixed with https://github.com/microsoft/winget-pkgs/pull/197015/commits/9c1ce25174ef6e68d4f1f6368a8d3866315f1a71

Reply:
This pull request still requires you to mark the conversation as resolved. 🙏